### DISCUSSION TITLE
Generate target platform change X86 issue

### DISCUSSION THREAD

**Discussion by pyking-wang** [AvaloniaUI/Avalonia#12227] [Q&A] [answered]
I set the target generation platform of the project from Any CPU to X86. 
After recompilation, avalonia outputs the following Error information and the preview process stops working. (After compilation, the program can run normally.)
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/215c01cb-12a9-43c7-a178-cd55fbffbb93)

**Comment by maxkatz6** [comment]
It's avalonia previewer error. It doesn't support x86 IIRC.

**Reply by pyking-wang** [reply]
Besides the preview, will it affect other functions？

**Reply by maxkatz6** [reply, marked as answer]
Avalonia does support x86 arch, but you usually need yo use "runtime identifier"  and not "platform"   

https://stackoverflow.com/questions/68825789/relationship-between-platformtarget-and-runtimeidentifier-in-net-5

**Reply by pyking-wang** [reply]
I see, thank you.(='◡'=)
